www/grpc/swagger-ui/pactus.swagger.json

```diff
@@ -128,7 +128,7 @@
         ]
       }
     },
-    "/v1/blockchain/block_hash/{height}": {
+    "/v1/blockchain/block_hash/height/{height}": {
```

Comment:
Please change height to heights in path

Reply:
```
"/v1/blockchain/block_hashes/height/{height}"
"/v1/blockchain/block_heights/hash/{hash}"
```
 or
 ```
 "/v1/blockchain/block_hash/heights/{height}"
 "/v1/blockchain/block_height/hashes/{hash}"
 ```

Reply:
How about this? @Ja7ad

Reply:
> How about this? @Ja7ad 

"/v1/blockchain/block_hashes/height/{height}"
"/v1/blockchain/block_heights/hash/{hash}"